pkg/metrics/metrics.go

```diff
@@ -37,28 +41,42 @@ var (
 		Name: "component_version",
 		Help: "Metric to expose the version of the PDCSI GKE component.",
 	}, []string{"component_version"})
+
+	pdcsiOperationErrorsMetric = metrics.NewGaugeVec(
```

Comment:
I don't think Gauge is what we want for errors. Counter fits better.

Reply:
What counter metrics are you referring to? Can you point the reference? It is difficult to go from the code editor to find what you meant without context. 

I have seen NewHistogramVec and NewGaugeVec so far.